plugin.php

```diff
@@ -4,7 +4,7 @@
  * Description: Advanced media tools that take advantage of Rekognition and Tachyon.
  * Author: Human Made Limited
  * License: GPL-3.0
- * Version: 0.4.3
+ * Version: 0.4.6
```

Comment:
This is not backwards compatible as we're removing and adding filters so I would have called it 0.5.0, but I don't know the project so feel free to ignore.

Reply:
I was thinking about that but it is backwards compatible in Altis terms. I just checked when 0.4.0 of this plugin was released though and that was in 2021 before WP 6.0 so you're right, it'll just take a bit longer to get the release out as we'll need an Altis patch too.

Reply:
ah, well this is no longer an altis plugin.